libraries/botframework-connector/tests/connector.test.js

```diff
@@ -4,420 +4,595 @@
 * license information.
 */
 
-var path = require('path');
+/*
+* WORKING WITH THESE TESTS
+*   To run mocked and without recording:
+*    1. Supply .env with:
+*       * NOCK_OFF=false
+*
+*   To run live or unmocked:
+*    1. Supply .env with (all must be valid): 
+*       * CLIENT_ID (appID)
+*       * CLIENT_SECRET (appPass)
+*       * USER_ID (from Slack)
+*       * BOT_ID (from slack)
+*       * AZURE_SUBSCRIPTION_ID
+*       * NOCK_OFF=true 
```

Comment:
I love this.

@mdrichardson do we need to update the build config in Azure DevOps for this added configuration? I'd love to have these tests run live nightly and for when we do RC builds.

Reply:
Assuming that by "run live", you mean unmocked, we'd need to add the environment variables to the build config. At least when committing this PR, it didn't seem that the test server used the .env file at all and instead fell back on defaults. You don't seem to need a slack-enabled bot running, but you do need a valid and existing appId/appPass that has the Slack channel enabled, so it would be good to use one that's somewhat "permanent"